src/main/java/com/acme/dbo/txlog/Facade.java

```diff
@@ -1,11 +1,35 @@
 package com.acme.dbo.txlog;
 
 public class Facade {
+    static String PrefixMessagePrimitive = "primitive: ";
+    static String PrefixMessageChar = "char: ";
+    static String PrefixMessageString = "string: ";
+    static String PrefixMessageReference = "reference: ";
+
+    private static void PrintMessage(String Richmessage) {
```

Comment:
first letter lower case for method naming in java )

Reply:
Fixed. Thanks.